qt_gui/src/qt_gui/dock_widget_title_bar.py

```diff
@@ -39,7 +39,7 @@ class DockWidgetTitleBar(QWidget):
 
     """Title bar for dock widgets providing custom actions."""
 
-    def __init__(self, dock_widget, qtgui_path):
+    def __init__(self, dock_widget, qtgui_path, hide_title):
```

Comment:
Please add the new argument as a keyword argument (`hide_title=False`) to avoid breaking public API.